### PR TITLE
Enabled react hot reloading

### PR DESCRIPTION
**Pull request by yash0087**
Updated the bootstrap config file in order for us to use the react hot loading functionality.
Now the webpack-dev-server runs in the user specified port or at 9876 and proxies all the request to the express server which runs at port 3000.

### CODE REVIEW

**Comment by romidane** [comment]
Can you use another port than 3000. It's too commonly used by other apps making its availability scarce.

File: src/preview-server/bootstrap.js [hunk 1]
```diff
@@ -1,83 +1,84 @@
...
+  PREVIEW_HOT_RELOAD_SERVER_PORT,
+  EXPRESS_SERVER_PORT
 } from '../constants';
 
 export default function bootstrap(options) {
   const COMPONENT_CONFIG = require(COMPONENT_CONF);
   const BLACKJACK_CONFIG = COMPONENT_CONFIG.blackjack || {};
   const PREVIEW_PATH = BLACKJACK_CONFIG.preview;
-  const FINAL_SERVER_PORT = options.port || PREVIEW_SERVER_PORT;
+  const FINAL_SERVER_PORT = options.port || PREVIEW_HOT_RELOAD_SERVER_PORT;
```
Comment (romidane) [inline src/preview-server/bootstrap.js]: This can be named better don't you think?

Comment (yash0087) [inline src/preview-server/bootstrap.js]: Ok. I have renamed PREVIEW_HOT_RELOAD_SERVER_PORT to WEBPACK_DEV_SERVER_PORT and have also updated the express server port to 8765

**Comment by mackstar** [comment]
Tested - works nicely! :shipit:

**Comment by joshnesbitt** [comment]
It looks like as a result of this PR we've lost the ability to pass `port` into the test command as described [here](https://github.com/sky-uk/blackjack-toolbelt/blob/master/src/application.js#L34)?

**Comment by mackstar** [comment]
Good spot @joshnesbitt will add this on the way home if I can.

**Comment by joshnesbitt** [comment]
Thanks @mackstar!

**Comment by yash0087** [comment]
Ah! I have kept the port same as the express server port which is hardcoded to 8765. I can change that to take the WEBPACK_DEV_SERVER_PORT ? Is that ok?

**Comment by joshnesbitt** [comment]
@yash0087 I think we just need to ensure it's a CLI option rather than `process.env.PORT` and we should be OK

**Pull request merged**